GitUI/CommandsDialogs/FormCheckoutRevision.cs

```diff
@@ -19,7 +19,7 @@ private FormCheckoutRevision()
         }
 
         public FormCheckoutRevision(GitUICommands commands)
-            : base(commands)
+            : base(commands, enablePositionRestore: true)
```

Comment:
Will we encounter the same effects as with FormCommit?

Reply:
A lot of dialogs have this on.

Thinking about this now, this should _probably_ be off.